plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorSmokeTest.java

```diff
@@ -79,4 +82,25 @@ public void testShowCreateTable()
                         format("   location = '.*/" + schemaName + "/region'\n") +
                         "\\)");
     }
+
+    @Test
+    public void testHiddenPathColumn()
+    {
+        String tableName = "hidden_file_column_table_metastore_" + randomTableSuffix();
+
+        assertUpdate("CREATE TABLE " + tableName + " (a int, b VARCHAR) ");
+        assertUpdate("INSERT INTO " + tableName + " VALUES (1, 'a')", 1);
+        assertQuery("SELECT a, b FROM " + tableName, "VALUES (1, 'a')");
+
+        MaterializedResult result = computeActual(format("SELECT file_path FROM \"%s$files\"", tableName));
+        assertEquals(result.getRowCount(), 1);
+        String filePath = (String) result.getOnlyValue();
+
+        assertQuery(format("SELECT DISTINCT \"$path\" FROM %s", tableName), format("VALUES " + "'" + filePath + "'"));
```

Comment:
please leave the format and use string concat here.

Reply:
done